edx/frontend-lib-content-components
# feat: problem editor settings

## Description

This PR adds the section to view and modify xblock settings in problem editor based on the [figma board](https://www.figma.com/file/eIYzYuPvyaFn5IijR3dFe3/Project-Board%3A-Content-Libraries-and-Editing?node-id=1130%3A25243)

![image](https://user-images.githubusercontent.com/36229857/193505987-8756b1b9-82f7-4b1f-b5f9-95d9de5cb29e.png)


Useful information to include:
- Which edX user roles will this change impact? Common user roles are "Learner", "Course Author",
"Developer", and "Operator".
- Include screenshots for changes to the UI (ideally, both "before" and "after" screenshots, if applicable).
- Provide links to the description of corresponding configuration changes. Remember to correctly annotate these
changes.

## Supporting information

Link to other information about the change, such as Jira issues, GitHub issues, or Discourse discussions.
Be sure to check they are publicly readable, or if not, repeat the information here.

## Testing instructions

1. Get the master branch working with LMS, Studio, and Course Authoring MFE and clone this repo in {DEVSTACK}/src and checkout this branch
2. There are two ways to view the work Gallery View and Studio
3. For studio to work add and activate waffle flag 'new_core_editors.use_new_problem_editor'
4. Now on studio add Problem and edit the problem, it should redirect to Course Authoring MFE.
5. The other way is to use Gallery View, get in the frontend-lib-content-components/www, run npm install, npm start.
6. Visit http://localhost:8080/index.html and you should see the problem rendered.

## Deadline

"None" if there's no rush, or provide a specific date or event (and reason) if there is one.

## Other information

Include anything else that will help reviewers and consumers understand the change.
- Does this change depend on other changes elsewhere?
- Any special concerns or limitations? For example: deprecations, migrations, security, or accessibility.
